README.md

```diff
@@ -96,6 +96,64 @@ class Test2 {
 } 
 ```
 
+#### Using custom inspections
+
+Scapegoat supports custom inspections, for example to enforce a
+project-specific rule.
+
+How to write and test the inspection class is described in [the scalac-scapegoat-plugin CustomInspections readme](https://github.com/sksamuel/scalac-scapegoat-plugin/blob/master/CustomInspections.md)
+
+Your inspections need to be compiled before the code in the rest of your project is compiled (like macros). You can achieve this with a project set up like the following:
+
+    object MyBuild extends Build {
+    
+      val scalaV = "2.11.6"
+    
+      lazy val project = Project("MyProject", file("."))
+        // "aggregate" causes these child projects to be built and tested whenever
+        // the parent project is built and tested.
+        .aggregate(inspections)
+        .settings(
+          scalaVersion := scalaV,    
+          // You must list here all the custom inspections by their fully-qualified class name:
+          scapegoatCustomInspections := List(
+            "my.inspections.InspectionOne",
+            "my.inspections.InspectionTwo"),
```

Comment:
(This comment is now in the codebase, see [below](https://github.com/sksamuel/sbt-scapegoat/pull/27/files#diff-5a749244d1e091ce0adda24da548f08dR17))